tests/add.bats

```diff
@@ -108,35 +108,62 @@ load helpers
 @test "add single file creates absolute path with correct permissions" {
   imgName=ubuntu-image
   createrandom ${TESTDIR}/distutils.cfg
+  permission=$(stat -c "%a" ${TESTDIR}/distutils.cfg)
 
   cid=$(buildah from --signature-policy ${TESTSDIR}/policy.json ubuntu)
   run_buildah add $cid ${TESTDIR}/distutils.cfg /usr/lib/python3.7/distutils
   run_buildah run $cid stat -c "%a" /usr/lib/python3.7/distutils
   root=$(buildah mount $cid)
-  expect_output "644"
+  expect_output $permission
   run_buildah commit --signature-policy ${TESTSDIR}/policy.json $cid containers-storage:${imgName}
   run_buildah rm $cid
 
   newcid=$(buildah from --signature-policy ${TESTSDIR}/policy.json ${imgName})
   run_buildah run $newcid stat -c "%a" /usr/lib/python3.7/distutils
-  expect_output "644"
+  expect_output $permission
   run_buildah rm $newcid
 }
 
 @test "add single file creates relative path with correct permissions" {
   imgName=ubuntu-image
   createrandom ${TESTDIR}/distutils.cfg
+  permission=$(stat -c "%a" ${TESTDIR}/distutils.cfg)
 
   cid=$(buildah from --signature-policy ${TESTSDIR}/policy.json ubuntu)
   run_buildah add $cid ${TESTDIR}/distutils.cfg lib/custom
   run_buildah run $cid stat -c "%a" lib/custom
   root=$(buildah mount $cid)
-  expect_output "644"
+  expect_output $permission
   run_buildah commit --signature-policy ${TESTSDIR}/policy.json $cid containers-storage:${imgName}
   run_buildah rm $cid
 
   newcid=$(buildah from --signature-policy ${TESTSDIR}/policy.json ${imgName})
   run_buildah run $newcid stat -c "%a" lib/custom
-  expect_output "644"
+  expect_output $permission
   run_buildah rm $newcid
 }
+
+@test "add with chown" {
+  createrandom ${TESTDIR}/randomfile
+  cid=$(buildah from --signature-policy ${TESTSDIR}/policy.json busybox)
+  run_buildah add --chown bin:bin $cid ${TESTDIR}/randomfile /tmp/random
+  run_buildah run $cid ls -l /tmp/random
+
+  expect_output --substring bin.*bin
+  buildah ps
+  buildah images
+  buildah rm $cid
+  buildah rmi busybox
+}
+
+@test "add url" {
+  cid=$(buildah from --signature-policy ${TESTSDIR}/policy.json busybox)
+  run_buildah add $cid https://github.com/containers/buildah/raw/master/README.md
+  run_buildah run $cid ls /README.md
+
+  run_buildah add $cid https://github.com/containers/buildah/raw/master/README.md /home
+  run_buildah run $cid ls /home/README.md
+
+  buildah rm $cid
+  buildah rmi busybox
```

Comment:
Why the rmi? It seems wasteful to force other tests to re-pull the image. It's also a great way to increase test runtime and increase risk of flakes due to network failures. I don't think we want that.

Reply:
OK. Will remove it. Just want to keep the test env clean and not influence other test cases. But yes, this is not necessary for every test. Will check them and remove the ones not needed. Thanks a lot.